cppwinrt/component_writers.h

```diff
@@ -508,6 +508,7 @@ catch (...) { return winrt::to_hresult(); }
                             type_name,
                             method_name,
                             bind<write_consume_params>(signature),
+                            is_put_overload(method) ? "" : "return ",
```

Comment:
Should be `is_return_type_void(method) ? "" : "return"` to cover methods returning void. For some imaginary helper function `is_return_type_void`.